includes/controls/groups/background.php

```diff
@@ -132,7 +132,7 @@ public function init_fields() {
 			'default' => '',
 			'title' => _x( 'Background Color', 'Background Control', 'elementor' ),
 			'selectors' => [
-				'{{SELECTOR}}' => 'background-color: {{VALUE}};',
+				'{{SELECTOR}}' => 'background: {{VALUE}};',
```

Comment:
Background color should work together with background image (for example if a PNG includes a transparent area)

Reply:
It works perfectly with a regular background color. 
I don't think there is a way to make it work with a gradient background without using JS.
What do you think? @matipojo